recipes/pytest-regtest/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pytest-regtest" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 97fd19d12c44a8697be51faca4904112ae6e8460e798ac37a7c5639179bfc2c1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - setuptools
+  run:
+    - pytest >=4.1.0
+    - python
```

Comment:
Can we leave a comment about the build versus run requirements?

Reply:
We checked again, and found out that we were wrong about the requirement of Python 3 for building. Therefore we removed it.